protocol/chainlib/grpc.go

```diff
@@ -384,11 +399,11 @@ func (cp *GrpcChainProxy) SendNodeMsg(ctx context.Context, ch chan interface{},
 			msgLocal := msgFactory.NewMessage(methodDescriptor.GetInputType())
 			err = proto.Unmarshal(nodeMessage.Msg, msgLocal)
 			if err != nil {
-				return nil, "", nil, err
+				return nil, "", nil, utils.LavaFormatError("Failed to unmarshal proto.Unmarshal(nodeMessage.Msg, msgLocal)", err)
```

Comment:
will this spam errors even if the send is a warning like in chain fetcher fails?

Reply:
we discussed it and agreed to keep it